src/main/java/com/readcsvfile/devapp/utils/CityRoutesUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.readcsvfile.devapp.utils;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.readcsvfile.devapp.controller.CSVController;
+import com.readcsvfile.devapp.model.CityRoutes;
+
+public class CityRoutesUtils {
+
+	private static List<CityRoutes> cityList= new ArrayList<>();
```

Comment:
This can be done using Map<String, List<CityRoutes>> so that retrieval of the data is faster compare to List.
For Example, {'JFK': [{All the cities associated with JFK}]}